launcher/src/local-storage-key-provider.ts

```diff
@@ -57,7 +58,7 @@ export class LocalStorageKeyProvider {
 
   async getPartOfPublicKey(file: string): Promise<string> {
     let content = await fs.readFile(file);
-    content = content.substring(content.indexOf(' ') + 1);
+    content = content.split(' ')[1];
```

Comment:
@batleforc could you please add a test for that?

Reply:
@batleforc It would be great to test it with a key like you mentioned in the issue.

Reply:
I will add the test, but i can't build the image to test it has a drop-in replacement in my env

Reply:
Test has been added